### DISCUSSION TITLE
Why is there a README_pypi.rst file?

### DISCUSSION THREAD

**Discussion by Spectre5** [robbievanleeuwen/section-properties#62] [Q&A] [answered]
Since ~2018, PyPI accepts README files in markdown.  Is there a specific reason that we still need this RST version or the README then, or can we drop it in favor of only having the markdown one?

See [here](https://packaging.python.org/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata)

Interestingly, the `setup.py` file already uses `long_description_content_type='text/markdown'` even though the `readme` function reads the reStructuredText version: `with open('README_pypi.rst') as f:` - which I'm slightly surprised even works.  Perhaps if PyPI automatically recognizes that the description is in RST then it just ignores the value in `long_description_content_type`?

**Comment by robbievanleeuwen** [comment, marked as answer]
Hi @Spectre5,

Yes, this was a bit of a hack I used when I first uploaded _sectionproperties_ to PyPI, which at the time, seemed to be the only way to get the logo to display on the PyPI page.

Hopefully this isn't required in 2021, will give the cleaner option a shot with the next release!

Robbie
